tests/ui/pages/data_entity/structure.ts

```diff
@@ -1,26 +1,27 @@
 import Button from '../../elements/button';
 import InputField from '../../elements/input-field';
+import List from '../../elements/list';
 import DataEntityPage from './data_entity.page';
 
 const SELECTORS = {
   structureTab: `[role="tab"]:has-text("Structure")`,
   submitButton: `[type="submit"]`,
-  columnString: name => `.MuiGrid-root.MuiGrid-container.css-1rwljbm:has-text('${name}')`,
   inputLabel: `[placeholder="Enter label name…"]`,
   inputDescription: `[name="internalDescription"]`,
   createNewLabelLink: `[role="presentation"]:has-text('Create new')`,
+  resultsList: `[aria-label="grid"]`,
+  listItem: `[role="rowgroup"] >> div`,
+  editButton: `button:has-text('Edit')`,
 };
 
 export default class StructurePage extends DataEntityPage {
-  async editColumn(name: string) {
-    await this.page
-      .locator(SELECTORS.columnString(name))
-      .locator('button', { hasText: 'Edit' })
-      .click();
+  async clickEditButton(listItemName: string) {
+    const listItem = this.resultsList.getListElement(listItemName);
+    await listItem.locator(SELECTORS.editButton).click();
```

Comment:
Can we please move this function definition to the bottom of the PO class? 

So getters will be always on the top. And function definition will be located after the getters.

NOTE:
- Pls have a look at `async search(text: string)` in another file